near-sdk-macros/src/lib.rs

```diff
@@ -436,7 +436,7 @@ pub fn derive_near_schema(input: TokenStream) -> TokenStream {
             #[allow(non_camel_case_types)]
             type #input_ident_proxy = #input_ident;
             {
-                use ::near_sdk::__private::schemars;
+                use ::near_sdk::schemars;
```

Comment:
Can we completely remove this `use` and use fully qualified path to schemars types?